src/PHPMailer.php

```diff
@@ -1554,7 +1554,7 @@ public function preSend()
                     $this->MIMEBody
                 );
                 $this->MIMEHeader = rtrim($this->MIMEHeader, "\r\n ") . static::$LE .
-                    static::normalizeBreaks($header_dkim) . static::$LE;
```

Comment:
Can we double check whether this would be better fixed by having `$header_dkim` *not* include an LE to start with, which is done on line 4671?

Reply:
Looking further into this - PHPMailer generally doesn't trust its own assembly of headers, and pretty much always strips breaks and readds them, for example in `smtpSend` and `getSentMIMEMessage()`. That said, `MIMEHeader` is expected to have a trailing break as it often gets `mailHeader` concatenated directly to it, and when we are DKIM signing, the signature header is appended to `MIMEHeader`, so it does need the trailing break here.